Bugsnag/Client/BugsnagClient.m

```diff
@@ -235,6 +232,12 @@ - (void)start {
 
     [self computeDidCrashLastLaunch];
 
+    if (self.configuration.telemetry & BSGTelemetryInternalErrors) {
+        BSGInternalErrorReporter.sharedInstance = [[BSGInternalErrorReporter alloc] initWithDataSource:self];
+    } else {
+        bsg_log_debug(@"Internal error reporting was disable in config");
```

Comment:
Minor (pre-existing) typo, probably not worth a CI re-run unless you have other changes:

```suggestion
        bsg_log_debug(@"Internal error reporting was disabled in config");
```

Reply:
Will fix in the release 👍